frontend/src/component/Account/PersonalData.jsx

```diff
@@ -73,35 +73,28 @@ export class PersonalData extends React.Component {
             />
           </div>
 
-          <div className='d-flex align-items-center flex-wrap'>
-            <input
-              className='personaldata__form__txtinput primaryColorBorderLighten form-control mt-3 mt-sm-0'
-              type='text'
-              data-cy='personaldata__form__txtinput__username'
-              placeholder={props.t('New username')}
-              value={state.newUsername}
-              onChange={this.handleChangeUserName}
-            />
+          <div className='d-flex flex-wrap'>
```

Comment:
please create a class for these attributes css. almost always prefer to create classes instead of use bootstrap elements